opentracing_instrumentation/client_hooks/requests.py

```diff
@@ -93,4 +102,16 @@ def host_port(self):
             return split_host_and_port(host_string=self.host_str,
                                        scheme=self.scheme)
 
-    requests.adapters.HTTPAdapter.send = send_wrapper
+
+patcher = RequestsPatcher()
+
+
+def set_patcher(custom_patcher):
+    global patcher
+    patcher = custom_patcher
+
+
+# install_patches method is deprecated,
+# kept for backward compatibility
```

Comment:
per the comment in __init__, let's remove these comments and keep the func as official API.

Reply:
These comments have been removed.